### ISSUE TITLE
Keep your King SAFE before doing anything

### ISSUE THREAD

**Issue by khanhtc1202** [khanhtc1202/til#65]
Today I had been baited by a really interesting puzzle

<img width="972" alt="Screen Shot 2021-01-26 at 20 09 35" src="https://user-images.githubusercontent.com/32532742/105838863-08e88280-6014-11eb-8acf-991e755da8f4.png">

At the very first glance, Bb5 pin black Queen with the King, then Nc7 to fork attack King and Queen is tempting 😂 

<img width="969" alt="Screen Shot 2021-01-26 at 20 22 05" src="https://user-images.githubusercontent.com/32532742/105839158-6a105600-6014-11eb-8eb4-9fc6ef1b76cc.png">

But the game was not that easy 🤣 , black has a brilliant move by Bb4+

<img width="979" alt="Screen Shot 2021-01-26 at 20 08 53" src="https://user-images.githubusercontent.com/32532742/105839235-890ee800-6014-11eb-988a-2a84a47f9f6e.png">

Could lead the game back to fair trading ( no advantage for white ) 😢 
This is the solution 👇 

https://user-images.githubusercontent.com/32532742/105839383-bbb8e080-6014-11eb-9ac4-4022d35f082a.mp4

The next will be, black loses Queen or Bishop ( and of course it's Bishop in this case )

The passing rate was `0.7%`, so I think it was not just me 😢 
To recap, keep in mind that __your king's safety is the first thing you have to consider before you make any move__ 📝

**Comment by khanhtc1202** [comment]
I got the same issue today 😢 
https://www.chess.com/puzzles/problem/1201396
